spread-dapp.html

```diff
@@ -194,9 +194,10 @@ <h5 class="card-title mb-3">Spread list:</h5>
 
   <!-- Template Main JS File -->
   <script src="assets/js/main.js"></script>
```

Comment:
```suggestion
  <script type="module" src="assets/js/main.js"></script>
```

Reply:
should we try to use the import statement to import modules? 
or is that for our next update? 👌

Reply:
I removed that because the code was not working properly when using import and modules, but now I tested it and it worked so I must have made a mistake before. I will add it back now. 😄